policy-controller/k8s/status/src/index.rs

```diff
@@ -6,6 +6,7 @@ use crate::{
 use ahash::{AHashMap as HashMap, AHashSet as HashSet};
 use chrono::offset::Utc;
 use chrono::DateTime;
+use gateway::RouteParentStatus;
```

Comment:
if this needs to be included i'd prefer to i'd prefer to include this on line 12 like:
```
use linkerd_policy_controller_k8s_api::{self as k8s, gateway::{self, RouteParentStatus}, ResourceExt};
```

having indirect/relative imports like this gets very confusing in my experience.